tests/filtered_relation/models.py

```diff
@@ -36,6 +36,8 @@ class Book(models.Model):
         related_query_name="book",
     )
     editor = models.ForeignKey(Editor, models.CASCADE)
+    number_editor = models.IntegerField(null=True)
```

Comment:
Probably redundant with `editor_number`. Should we remove it?

Tests with `editor_number` would still fail if we only checked `if child.name.startswith(prefix)`, without the `__` lookup separator.

Reply:
It's probably worth keeping both.